test/integration/default/serverspec/spec_helper.rb

```diff
@@ -1,12 +1,17 @@
+require 'busser/rubygems'
+Busser::RubyGems.install_gem('rest-client', '~> 2.0')
```

Comment:
this is still strange to me that I need this because my understanding is busser uses the same ruby as chef-client (from the omnibus installer). Yet, I was unable to load `rest-client` in specs without this.